packages/@aws-cdk/aws-applicationautoscaling/test/test.scalable-target.ts

```diff
@@ -85,6 +85,40 @@ export = {
     test.done();
   },
 
+  'add scheduled scaling with cfnParameter'(test: Test) {
```

Comment:
same as my other comment. Feels like the test in test.cron.ts should provide sufficient cover. Do we need an extra test case?

Reply:
Sure, let me check and remove if not required.